src/lib/spacefinder/article-body-adverts.ts

```diff
@@ -62,18 +60,6 @@ let insertedDynamicAds: Advert[] = [];
 const getStickyContainerClassname = (i: number) =>
 	`${adSlotContainerClass}-${i + 2}`;
 
-const wrapSlotInContainer = (
```

Comment:
not blocking, but I think we could do this in `liveblog-adverts.ts` too?

Reply:
great shout, will sort that out separately.